web/styles/zulip.css

```diff
@@ -2168,6 +2168,61 @@ body:not(.spectator-view) {
         line-height: 20px;
         white-space: normal;
     }
+
+    &.compose_select_recipient-dropdown-list-container {
+        .dropdown-list .dropdown-list-item-common-styles {
+            /* We align items to the baseline for the sake of
+               the compose_select_recipient dropdown, which has
+               icons not present in other dropdowns. We also want
+               to hold the icon to the first baseline. */
+            align-items: first-baseline;
```

Comment:
Linter thinks this isn't a thing; was it meant to be `align-items: first baseline`?